components/rmrk/Gallery/GalleryItem.vue

```diff
@@ -234,18 +234,20 @@ export default class GalleryItem extends mixins(PrefixMixin) {
 
   public mounted() {
     // used to poll nft every second after component initialization in order to prevent double spending
-    this.$apollo.addSmartQuery<{ nft }>('nft', {
-      client: this.urlPrefix,
-      query: nftByIdMini,
+    this.$apollo.addSmartQuery<{ nftEntities }>('nft', {
```

Comment:
what's wrong here?
I replace `nftByIdMini` with `nftEntitiesByIDs`, so that I could fetch events with 'currentOwner' from 'subsquid' 006 .

Reply:
Because `nftByIdMini` is just a critial function to prevent double spend on RMRK NFTs :)

Reply:
I know that it needs to fetch the fresh data every few seconds. After this change, I think it will do the same thing.

Reply:
> After this change, I think it will do the same thing.

yup but bit slower :) with more overhead

if you need to update events I watch for the change of `owner/price/burned`

Reply:
> I watch for the change of owner/price/burned
I'm not sure what does it mean? Could you pls expand more? Do you mean that to simplify the data in the query?

Reply:
Oh, I see.

Reply:
please revert to the original state